lang/cs/Org.Apache.REEF.Client/Common/IApplicationReport.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using Org.Apache.REEF.Client.YARN.RestClient.DataModel;
+using Org.Apache.REEF.Utilities.Attributes;
+
+namespace Org.Apache.REEF.Client.Common
+{
+    /// <summary>
+    /// This interface represents application information mainated by Yarn RM.
+    /// This interface is modeled on Org.Apache.REEF.Client.YARN.RestClient.DataModel.Application. 
+    /// Documentation on the class Org.Apache.REEF.Client.YARN.RestClient.DataModel.Application
+    /// can be found here.
+    /// <a href="http://hadoop.apache.org/docs/r2.6.0/hadoop-yarn/hadoop-yarn-site/WebServicesIntro.html">
+    /// Hadoop RM REST API</a> documentation.
+    /// </summary>
+    [Unstable("0.16", "Working in progress. For local runtime, some of the property, such as FinalState and AppId are not implemented yet.")]
+    public interface IApplicationReport
+    {
+        /// <summary>
+        /// Get Yarn application name.
+        /// </summary>
+        string AppName { get; }
+        
+        /// <summary>
+        /// Get Yarn application start time.
+        /// </summary>
+        long StartedTime { get; }
+
+        /// <summary>
+        /// Get Yarn application finish time.
+        /// </summary>
+        long FinishedTime { get; }
+
+        /// <summary>
+        /// Get Yarn application number of running containers.
+        /// </summary>
+        int RunningContainers { get; }
```

Comment:
If this is in the `common` part of the REEF API, it should use REEF notations in the method names and comments. For instance, this should be called `NumberOfRunningEvaluators` or such.